### PR TITLE
PRO-1262:  Formatted Fiat Value for gas estimates lower then 0.01$

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1262/formatted-fiat-value-for-gas-estimates-lower-then-001dollar

## Description

- Show <$0.01 for less than 0.01 fiat value

## How Has This Been Tested?

- Tested on Simulator 12

## Screenshots (if appropriate):

https://user-images.githubusercontent.com/82652040/231667715-ee6f7545-3da3-4001-a6ec-4071df46500e.mp4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/components/FeeLabelToggle/FeeLabelToggle.tsx [hunk 1]
```diff
@@ -96,7 +96,8 @@ const FeeLabelToggle: FC<IFeeLabelToggle> = ({
   const currencySymbol = getCurrencySymbol(fiatCurrency);
 
   const feeInFiat = getTxFeeInFiat(chain, txFeeInWei, gasToken, chainRates, fiatCurrency);
-  const feeInFiatDisplayValue = `${currencySymbol}${feeInFiat.toFixed(2)}`;
+  const feeInFiatDisplayValue =
+    feeInFiat < 0.01 && feeInFiat > 0 ? `<${currencySymbol}0.01` : `${currencySymbol}${feeInFiat.toFixed(2)}`;
```
Comment (MayukhSamadder2000) [inline src/components/FeeLabelToggle/FeeLabelToggle.tsx:100]: just thinking will `< ${currencySymbol}0.01` be better? Just added a space between < and the amount

File: src/utils/common.js [hunk 2]
```diff
@@ -316,7 +316,9 @@ export const formatFiatValue = (value: Value, options?: { skipCents?: boolean })
 
 export const formatFiat = (value: Value, baseFiatCurrency?: ?string, options?: { skipCents?: boolean }): string => {
   const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
-  return `${getCurrencySymbol(fiatCurrency)}${formatFiatValue(value, options)}`;
+  return parseFloat(value) < 0.01 && parseFloat(value) > 0
+    ? `<${getCurrencySymbol(fiatCurrency)}0.01`
```
Comment (MayukhSamadder2000) [inline src/utils/common.js:320]: same as above.

File: src/components/FeeLabelToggle/FeeLabelToggle.tsx (hunk 1 above)
Comment (lalitkumarrock) [reply src/components/FeeLabelToggle/FeeLabelToggle.tsx]: @MayukhSamadder2000 Updated it.

File: src/utils/common.js (hunk 2 above)
Comment (lalitkumarrock) [reply src/utils/common.js]: @MayukhSamadder2000 Updated it.

**Review by Yash-Chaubal — APPROVED** [review]
LGTM!

**Pull request merged**